Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Tuxemon/Tuxemon](https://hosted.weblate.org/projects/tuxemon/tuxemon/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/tuxemon/-/tuxemon/horizontal-auto.svg)

cyphra committed:
Translated using Weblate (Spanish)

Currently translated at 71.8% (925 of 1288 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/es/

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -259,7 +260,7 @@ msgid "generic_failure"
 
 msgid "attempting_capture"
-msgstr "Intentando capturar…"
+msgstr "Intentando capturar..."
```
Why changing the Unicode character by three dots? Weblate seems to prefer the former. I think that at least we should discuss it and reach a consensus between all translation groups.

bitcraft replied:
lets stick to three dots for now

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -1102,7 +1103,7 @@ msgstr "Zunna"
 # Monster categories
 msgid "anteater"
-msgstr "Oso hormiguero"
+msgstr "Amenaza"
```
What?

cyphra replied:
Several translation avaliable

vnmabus replied:
I don't see any other translation. Plus, from the context it is clear that it is "oso hormiguero", as it is the category of Aardart (https://www.fakemon.net/fakemon/fakemon1595937788DGN.png).

vnmabus replied:
```suggestion
msgstr "Oso hormiguero"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -1613,11 +1635,11 @@ msgstr "No me queda mucho tiempo. Omnichannel está viniendo...\n"
 
 msgid "quicklyknight3"
-msgstr "Allie. El nombre es Allie."
+msgstr "Allie. Allie es el que no quiere que sepas algo."
```
I think Allie is female. Also "sepas algo" is too ambiguous. It does not look realistic.
```suggestion
msgstr "Allie. Allie es la que no quiere que lo sepas."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -1687,7 +1713,9 @@ msgid "cometomepls"
 msgid "hellolovelypeople2"
-msgstr "Sé que probablemente os preguntáis por qué mis Omni caballeros y yo estamos aquí.\n"
+msgstr ""
+"Sé que probablemente os preguntáis por qué mis Caballeros Omni y yo estamos "
```
I think "Omni caballeros" sounds better. Maybe it is better with an hyphen, or joined as one word.

cyphra replied:
Usually the translation change the order

vnmabus replied:
Translation is not a mechanical thing. When we have the choice we have to pick the option that sounds better in the target language. "Omni" in Spanish is always a prefix, so the best sounding words would be "Omnicaballeros", "Omni-Caballeros" or "Omni Caballeros". I think I prefer the first one: "Omnicaballeros". What do you think?

vnmabus replied:
```suggestion
"Sé que probablemente os preguntáis por qué mis Omnicaballeros y yo estamos "
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -1789,7 +1820,8 @@ msgstr "Cariño, espera. Acabo de encender la televisión y echaban una rueda de
 msgid "ipausedit"
-msgstr "La he puesto en pausa para poder coger un vaso de agua y verla juntos."
+msgstr ""
+"He puesto la TV en pausa, para poder coger un vaso de agua y verla juntos."
```
I think either the previous translation or the following one are more idiomatic (nobody says TV aloud in Spanish):
```suggestion
"He pausado la tele para poder coger un vaso de agua y verla juntos."
```

cyphra replied:
Yep. I prefer "la tele" 👍🏽

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -1849,7 +1881,7 @@ msgid "okaythen"
 
 msgid "okaythen2"
-msgstr "Bien, ¡parece que tus Tuxemon están llenos de energía! ¡Aquí están!"
+msgstr "Bien, ¡parece que tus Tuxemon están mejor ahora ¡Aquí están!"
```
```suggestion
msgstr "Bien, ¡parece que tus Tuxemon están mejor ahora! ¡Aquí están!"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -1950,10 +1982,15 @@ msgstr "<recovery_module>: GREETINGS ${{name}}. PLEASE UNDERSTAND THIS AREA NEVE
+"extraño viniendo de mí, pero necesito saberlo...\n"
+"Blank: ¿Sabes quién soy? ¿Me has... visto alguna vez? Tengo curiosidad: mi "
+"nombre, ¿tengo un...\n"
+"Blank: No. Lo siento: he cometido un error. Por favor, no le diga a ninguno "
+"que le pregunté. Continúa, disfruta tu viaje aquí."
```
I think the older translation, although not literally the same as the original, is more idiomatic. In either case, "usted" is rarely used in peninsular Spanish, especially with young people:

```suggestion
"Blank: No. Lo siento: he cometido un error. Por favor, no le digas a nadie "
"que te he preguntado eso. Continúa, disfruta tu viaje aquí."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -2455,15 +2493,15 @@ msgstr ""
 "Desde que nos cortaron el presupuesto hemos tenido que vender las obras de "
 "arte en lugar de solo mostrarlas.\n"
-"La entrada cuesta 50${{currency}}. Va a los dividendos anuales de la compañía."
+"La entrada cuesta 50$. Va a los dividendos anuales de la compañía."
```
You should use the old translation. `${{currency}}` will be replaced with the proper money symbol of the campaign, with may be different from the dollar sign.

vnmabus replied:
```suggestion
"La entrada cuesta 50${{currency}}. Va a los dividendos anuales de la compañía."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -2455,15 +2493,15 @@ msgstr ""
+" Desde que nos cortaron el presupuesto hemos tenido que vender las obras de "
 "arte en lugar de solo mostrarlas.\n"
-"La entrada es 50${{currency}}. Va a los dividendos anuales de la compañía."
+" La entrada es 50$. Va a los dividendos anuales de la compañía."
```
Same as above.

vnmabus replied:
```suggestion
" La entrada es 50${{currency}}. Va a los dividendos anuales de la compañía."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -2565,7 +2603,7 @@ msgstr ""
 msgstr ""
 "He oído que están haciendo un nuevo parque. Para \"crear espacios verdes en "
-"la zona\"."
+"la zona\""
```
Why did you remove the period?

vnmabus replied:
```suggestion
"la zona\"."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
```
The passive here is slightly strange, but passable.

```suggestion
msgstr "Cuando fui emboscado en la duna por un SPIGHTER estuve muy asustado."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
```
That is plain wrong. This sentence is a pun on the previous one, with talks about tennis. The pun should be preserved somehow, not translated literally.

vnmabus replied:
I don't know how to preserve the same meaning and the relation with the previous sentence. The best thing I can come with is
```suggestion
msgstr "El tenis se me da mejor..."
```
that means "I am better at tennis...". @cyphra, @Sanglorian, what do you think?

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Me sirvieron."
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
```
I am not sure about the context of this one. Is this translation appropriate?

vnmabus replied:
```suggestion
msgstr "¡Qué mala suerte!"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "¡Gorrón!"
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
```
"creeped" is not "arrastrado" but "asustado". It makes more sense if you look at the previous sentence in the dialog.
```suggestion
msgstr "Oye, ¡no parecías asustado en absoluto!"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
+
+msgid "menu_down_key"
+msgstr "Tecla Bajar"
```
Nobody says that in Spanish. It could be "Flecha Abajo" or better, simply "Abajo" (which even makes sense for a controller).
```suggestion
msgstr "Abajo"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Tecla Bajar"
+
+msgid "menu_secondary_select_key"
+msgstr "Tecla Cancelar"
```
```suggestion
msgstr "Botón Cancelar"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Tecla Cancelar"
+
+msgid "spyder_route4_wulf2"
+msgstr "Ooh, ¡me siento tan castrado!"
```
NOBODY says this in Spanish. You shouldn't translate literally.

vnmabus replied:
```suggestion
msgstr "Ooh, ¡me siento tan débil!"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Ooh, ¡me siento tan castrado!"
+
+msgid "menu_left_key"
+msgstr "Tecla Izquierda"
```
```suggestion
msgstr "Izquierda"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Tecla Izquierda"
+
+msgid "dark_robo"
+msgstr "Robo Oscuro"
```
What is the context? Is this a robot? Because "robo" is "theft" in Spanish, so maybe we should use a different word.

vnmabus replied:
```suggestion
msgstr "Robot Oscuro"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Robo Oscuro"
+
+msgid "spyder_citypark_maniac"
+msgstr "Phew, ¡lo hemos cagado!"
```
Nobody says "phew" in Spanish.
```suggestion
msgstr "Puf, ¡la hemos cagado!"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Phew, ¡lo hemos cagado!"
+
+msgid "dodge_increased"
+msgstr "el {stat} de {user} ha incrementado."
```
"incrementar" is transitive. Also they may be masculine and feminine stats, so using "el" and "incrementarse" (which should is conjugated different for each gender) is not always appropriate.

```suggestion
msgstr "{stat} de {user} ha aumentado."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "el {stat} de {user} ha incrementado."
+
+msgid "menu_up_key"
+msgstr "Tecla Subir"
```
```suggestion
msgstr "Arriba"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Tecla Subir"
+
+msgid "menu_right_key"
+msgstr "Tecla Derecha"
```
```suggestion
msgstr "Derecha"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Tecla Derecha"
+
+msgid "menu_primary_select_key"
+msgstr "Tecla Acción"
```
```suggestion
msgstr "Botón Acción"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Tecla Acción"
+
+msgid "menu_back_key"
+msgstr "Tecla Volver"
```
Not very sure about this one...
```suggestion
msgstr "Botón Retorno"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Tecla Volver"
+
+msgid "options_new_input_key0"
+msgstr "Pulse una tecla para fijar su tecla de entrada nueva"
```
Too literal. I don't like the keyboard assumption here, but it is also in the original sentence:

```suggestion
msgstr "Pulsa una tecla para establecerla como nuevo valor de la entrada."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Pulse una tecla para fijar su tecla de entrada nueva"
+
+msgid "options_new_input_key1"
+msgstr "Reinicie su juego para que los cambios surtan en efecto."
```
```suggestion
msgstr "Reinicia el juego para que los cambios surtan efecto."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+msgstr "Xeón"
+
+msgid "spyder_route4_wulf1"
+msgstr "Ah, tan solo recoger algunos suplementos medicinales"
```
Use the context. This is the first sentence that a trainer says!

```suggestion
msgstr "Ah, solo estaba recogiendo algunos suplementos medicinales."
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+
+msgid "spyder_wayfarer1_bismuth1"
+msgstr ""
+"¡Espero que no esté relacionado a aquellos tuxemon enfermos en las granjas "
+"Scoop!"
```
```suggestion
"¡Espero que no esté relacionado con aquellos Tuxemon enfermos en las granjas "
"Scoop!"
```

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
+"Scoop!"
+
+msgid "hp_restored"
+msgstr "el hp del {user} ha sido recuperada"
```
HP is hit points. In Spanish it would be "puntos de golpe" or simply "salud" (health). You also are mixing genders here.
```suggestion
msgstr "La salud de {user} ha sido restaurada."
```

vnmabus reviewed (changes requested):
Thank you for your contribution @cyphra ! I have several comments and suggestions. If you want to further contribute to Tuxemon's translation to Spanish we can coordinate the translation in the future using the translation channel in Tuxemon's Discord.

Commit:
Translated using Weblate (Spanish (Mexico))

Currently translated at 37.8% (487 of 1288 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/es_MX/

Commit:
Translated using Weblate (Spanish (Mexico))

Currently translated at 41.8% (539 of 1288 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/es_MX/

Commit:
Translated using Weblate (Spanish (Mexico))

Currently translated at 50.9% (656 of 1288 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/es_MX/

Commit:
Translated using Weblate (Spanish (Mexico))

Currently translated at 51.0% (657 of 1288 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/es_MX/

Commit:
Translated using Weblate (Spanish (Mexico))

Currently translated at 52.1% (672 of 1288 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/es_MX/

Commit:
Translated using Weblate (Spanish (Mexico))

Currently translated at 52.5% (677 of 1288 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/es_MX/

vnmabus commented on mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:
```diff
@@ -1634,13 +1656,16 @@ msgstr "Sí, señora."
 msgid "bothofyou"
-msgstr "Vosotros dos, venid fuera. Un contingente de Omnichannel requiere vuestra presencia."
+msgstr ""
+"Vosotros dos, venid fuera. Una representación de Omnichannel desea dirigirse "
```
```suggestion
"Vosotros dos, venid fuera. Un representante de Omnichannel desea dirigirse "
```

bitcraft committed:
Update base.po

suggestions

bitcraft committed:
Apply suggestions from code review

Co-authored-by: Carlos Ramos Carreño <carlosramosca@hotmail.com>

bitcraft committed:
Apply suggestions from code review

Co-authored-by: Carlos Ramos Carreño <carlosramosca@hotmail.com>

vnmabus commented:
I think you can apply the last suggestion and merge it.

bitcraft committed:
Update mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

Co-authored-by: Carlos Ramos Carreño <carlosramosca@hotmail.com>
